arch/c4_context.md

```diff
@@ -6,40 +6,70 @@ All production Systems (External and Radix) and Users that are commonly found on
 
 ![c4_context.puml](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://raw.githubusercontent.com/radixdlt/docs/arch/arch/c4_context.puml)
 
+# Universes
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/).
+
+From an architectural point of view, a Universe identifies and overlay network on top of Internet or in a LAN.
+Core/Boot nodes can be configured to be part of a specific universe.
+
+# Atoms and Particles
+
+Described in detail in the Radix [Whitepaper v2](https://papers.radixdlt.com/tempo/v2/).
+
+On an Architectural level an Atom is totally ordered sequence of Transactions (Particles) that must be validated as a unit and committed to the Ledger (DB) atomically. 
+
+The minimal unit of transactional data transferred over wire (sync/gossip/WebSocket) is thus an Atom.
```

Comment:
transferred over wire *and* persisted in nodes - nodes only store atoms, not partial atoms (particles or whatevs)